code/modules/clothing/accessories/holomap.dm

```diff
@@ -364,15 +364,20 @@ var/list/holomap_cache = list()
 		holomap_images += bgmap
 
 /obj/item/clothing/accessory/holomap_chip/proc/update_marker()
+	marker_prefix = base_prefix
 	if (prefix_update.len > 0)
-		marker_prefix = base_prefix
 		var/obj/item/clothing/under/U = attached_to
 		if(U && ishuman(U.loc))
 			var/mob/living/carbon/human/H = U.loc
 			var/obj/item/helmet = H.get_item_by_slot(slot_head)
 			if(helmet && "[helmet.type]" in prefix_update)
 				marker_prefix = prefix_update["[helmet.type]"]
 
+/obj/item/clothing/accessory/holomap_chip/raider/update_marker()
+	..()
+	if(rand(100) > 90)
```

Comment:
I had forgotten about prob(). I'm re-learning C# right now so new methods are overwriting the old ones in my brain.